lib/vega_lite/data.ex

```diff
@@ -205,9 +205,9 @@ defmodule VegaLite.Data do
   list of fields to be encoded.
 
   As a specialized chart, the jointplot expects an `:x` and `:y` and optionally a `:color` and a
-  `:text` field. All data must be `:quantitative` and the default `:kind` is `:circle`.
+  `:text` field. All data must be `:quantitative`.
 
-  It also accepts the specialized `:density_heatmap` as `:kind`.
+  Besides all marks, it also accepts the specialized `:density_heatmap` as kind.
```

Comment:
Let's call the argument `mark` to avoid confusion, especially because it can also be a list. And let's have an example of it being a list too. And then we say "It supports a special mark called `density_heatmap`".